src/main/java/club/sk1er/hytilities/command/HytilitiesCommand.java

```diff
@@ -1,32 +1,267 @@
 package club.sk1er.hytilities.command;
 
 import club.sk1er.hytilities.Hytilities;
+import club.sk1er.hytilities.handlers.chat.customrestyle.CustomRestyle;
+import club.sk1er.hytilities.handlers.chat.customrestyle.RecievedMessageFormat;
 import club.sk1er.mods.core.ModCore;
+import com.google.gson.JsonParseException;
 import net.minecraft.command.CommandBase;
 import net.minecraft.command.ICommandSender;
+import net.minecraft.util.BlockPos;
+import org.jetbrains.annotations.NotNull;
+
+import java.io.IOException;
+import java.util.*;
+import java.util.regex.Pattern;
+
+import static club.sk1er.hytilities.Hytilities.sendMessage;
 
 public class HytilitiesCommand extends CommandBase {
+
+    private final Map<String, RecievedMessageFormat> formats = new HashMap<>();
+
     @Override
     public String getCommandName() {
-        return "hytilities";
+        return Hytilities.MOD_ID;
```

Comment:
this isn't necessary, the command name nor modid will never be changed

Reply:
I see no issue changing this however. This is the correct implementation.